gobblin-core/src/main/java/gobblin/source/extractor/hadoop/HadoopFsHelper.java

```diff
@@ -0,0 +1,95 @@
+package gobblin.source.extractor.hadoop;
+
+import gobblin.configuration.ConfigurationKeys;
```

Comment:
Wrong order of imports, see https://github.com/linkedin/gobblin/wiki/CodingStyle.